pkgs/tools/text/rpl/default.nix

```diff
@@ -1,22 +1,41 @@
-{ lib, fetchFromGitHub, pythonPackages }:
+{ lib, fetchFromGitHub, python3Packages }:
 
-pythonPackages.buildPythonApplication rec {
+let
+  rev = "4467bd46a7a798f738247a7f090c1505176bd597";
+  sha256 = "0yf3pc3fws4nnh4nd8d3jpglmsyi69d17qqgpcnkpqca5l4cd25w";
+in
+
+python3Packages.buildPythonApplication rec {
   pname = "rpl";
-  version = "1.5.7";
+  version = builtins.substring 0 7 rev;
```

Comment:
```suggestion
  version = "1.10";
```

Reply:
there is no tag pushed to the repo apparently but according to commit message that should be it + it's more human friendly